ui/hooks/useAssetDetails.js

```diff
@@ -60,6 +66,7 @@ export function useAssetDetails(tokenAddress, userAddress, transactionData) {
     collectibles,
     tokensWithBalances,
     prevTokenBalance,
+    isTokenCategory,
   ]);
 
   if (currentAsset) {
```

Comment:
Does this hook has existing other usages also ?

Reply:
Yes, it is used in <ConfirmTokenTransactionSwitch/>  as well. Since `useTransactionDisplayData` hook is not token-specific, it was throwing an error `'Unable to detect valid token data` while calling `getAssetDetails` from `useAssetDetails` while parsing the txData. So I added an `isTokenCategory` check to ensure only token approve and token transfer activities will call `getAssetDetails`.